client/scripts/views/components/sidebar/tag_selector.ts

```diff
@@ -50,7 +51,7 @@ const TagRow: m.Component<ITagRowAttrs, {}> = {
         m('span.tag-name', name),
       ],
       contentRight: [
-        !hideEditButton && m('.tag-count', pluralize(count, 'post')),
+        !hideEditButton && count && m('.tag-count', pluralize(count, 'post')),
```

Comment:
Would this make more sense as `count !== null`?

Reply:
it could be, but if a tag goes from having one thread to no threads, it should also not render a count, no?

Reply:
Looking at the design now I think it looks fine as is!